cmd/pilot-agent/BUILD

```diff
@@ -7,6 +7,7 @@ go_library(
     visibility = ["//visibility:private"],
     deps = [
         "//cmd:go_default_library",
+        "//platform/consul:go_default_library",
```

Comment:
by k8s to you mean a dependency on the istio pilot/platform/kube package or a dependency on the external kubernetes github repo?  Either way, right now agent is just using a utility function in pilot's platform/consul package, so I could move the util function into another common package in pilot